server/http/restapi/restapi.go

```diff
@@ -43,7 +43,7 @@ func Init(router *mux.Router, mm *pluginapi.Client, conf config.Service, proxy p
 	subrouter.HandleFunc(config.KVPath+"/", a.handleKV(a.kvList)).Methods("GET")
 	subrouter.HandleFunc(config.KVPath+"/{key}", a.handleKV(a.kvHead)).Methods("HEAD")
 	subrouter.HandleFunc(config.KVPath+"/{key}", a.handleKV(a.kvDelete)).Methods("DELETE")
-
+	subrouter.HandleFunc(config.Webhook+"/{app_id}/{path}", a.handleWebhook).Methods("POST")
```

Comment:
this should all go into the `gateway` package for now, with the route `[plugin url]/apps/appid/webhooks`